slack/src/test/groovy/org/togglz/slack/notification/MarkdownSpec.groovy

```diff
@@ -0,0 +1,47 @@
+package org.togglz.slack.notification
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+@Unroll
```

Comment:
https://github.com/szpak/spock-global-unroll could be used to unroll all parameterized tests in the whole module